temporalio/contrib/opentelemetry.py

```diff
@@ -158,8 +158,13 @@ def _start_as_current_span(
         *,
         attributes: opentelemetry.util.types.Attributes,
         input: Optional[_InputWithHeaders] = None,
+        kind: opentelemetry.trace.SpanKind = opentelemetry.trace.SpanKind.INTERNAL
```

Comment:
```suggestion,
        kind: opentelemetry.trace.SpanKind,
        input: Optional[_InputWithHeaders] = None
```

Lets make this required